schemas/gen.py

```diff
@@ -382,10 +410,12 @@ def go_unions(
     # Define getters for each of the common members of the union.
     common_members = get_union_common_members(file, package, union_types)
     for common_field, type in common_members:
-        lines.append(f"func ({x} {struct}) Get{common_field}() {type} {{")
+        # Getter.
+        getter = common_field[len("Raw") :]
+        lines.append(f"func ({x} {struct}) {getter}() {type} {{")
         for field, _ in union_spec:
             lines.append(f"\tif {x}.{field} != nil {{")
```

Comment:
use a switch 🤣... jk..

Reply:
You laugh but I expect this will happen if even partial linting is enabled.